### PR TITLE
Update parsers to return extra metadata info

### PR DESCRIPTION
**Pull request by ted-palmer**

### CODE REVIEW

File: src/parsers/onchain.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ export const parse = (asset) => {
     // so by default we ignore them (this behaviour can be overridden if needed).
     description: asset.description || null,
     imageUrl: normalizeLink(asset?.image) || null,
+    originalImageUrl: asset?.image || null,
```
Comment (nofir) [inline src/parsers/onchain.js:14]: I think you would still need to normalize , otherwise its not a valid url

**Review by nofir — CHANGES_REQUESTED** [review]

File: src/parsers/rarible.js
Comment (ptrwtts) [inline src/parsers/rarible.js:1]: definitely don't need to worry about rarible :) can consider it deprecated

File: src/parsers/simplehash.js [hunk 2]
```diff
@@ -9,7 +9,12 @@ export const parse = (asset) => {
     // Token descriptions are a waste of space for most collections we deal with
     // so by default we ignore them (this behaviour can be overridden if needed).
     description: asset.description,
+    originalMetadata: JSON.stringify(asset),
     imageUrl: asset.previews?.image_medium_url ?? asset.image_url,
+    imageOriginalUrl: asset.extra_metadata.image_original_url,
+    animationOriginalUrl: asset.extra_metadata.animation_original_url,
+    metadataOriginalUrl: asset.extra_metadata.metadata_original_url,
+    imageProperties: asset.image_properties,
```
Comment (nofir) [inline src/parsers/simplehash.js:17]: What is this? Is it only supported by simplehash

**Review by nofir — COMMENTED** [review]
Looks good, left one comment

File: src/parsers/simplehash.js (hunk 2 above)
Comment (ted-palmer) [reply src/parsers/simplehash.js:17]: Are you referring to image_properties? Peter mentioned he wanted to start saving that info: [https://reservoir0x.slack.com/archives/C0582JG5BMM/p1684470237526469](https://reservoir0x.slack.com/archives/C0582JG5BMM/p1684470237526469). It just has some basic info about the image type, width and height

**Review by nofir — APPROVED** [review]

**Pull request merged**